lnd.go

```diff
@@ -182,10 +181,9 @@ func lndMain() error {
 	}
 	sCreds := credentials.NewTLS(tlsConf)
 	serverOpts := []grpc.ServerOption{grpc.Creds(sCreds)}
-	grpcEndpoint := fmt.Sprintf("localhost:%d", loadedConfig.RPCPort)
-	restEndpoint := fmt.Sprintf(":%d", loadedConfig.RESTPort)
-	cCreds, err := credentials.NewClientTLSFromFile(cfg.TLSCertPath,
-		"")
+	grpcEndpoint := cfg.RPCListeners[0]
+	restEndpoint := fmt.Sprintf(":%d", cfg.RESTPort)
```

Comment:
Arguable that for the REST endpoint, it should listen on the same interfaces as the regular RPC interface. In the future if users request to have them listening on diff interfaces, we can add an option for that as well. Up to you which path to take for now...

Reply:
I went ahead and added `--restlisten` just to be consistent.